docs/gallery/External_Resources.py

```diff
@@ -37,24 +37,30 @@
 # as a string, but if you want to reference a key then you reference
 # the key object itself.
 
-data = Data(name="species", data='homo sapien')
-er.add_ref(container=data, field='Data', key='human', resource_name='NCBI_Taxonomy',
+data = Data(name="species", data=['Homo sapien', 'Mus Musculus'])
+er.add_ref(container=data, field='', key='Homo sapien', resource_name='NCBI_Taxonomy',
            resource_uri='https://www.ncbi.nlm.nih.gov/taxonomy', entity_id='NCBI:txid9606',
            entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9606')
 
-# The data is stored in KeyTable, ObjectTable, EntityTable, and ResourceTable.
+er.add_ref(container=data, field='', key='Mus Musculus', resource_name='NCBI_Taxonomy',
+           resource_uri='https://www.ncbi.nlm.nih.gov/taxonomy', entity_id='NCBI:txid10090',
+           entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=10090')
 
-# This will return a pandas dataframe of the ResourceTable
-er.resources.to_dataframe()
+###############################################################################
+# Using the get_keys method
+# ------------
```

Comment:
Minor, but Sphinx will complain if the header line is shorter than the header text
```suggestion
# ---------------------------------------------------------------
```